contracts/src/FeeContract.sol

```diff
@@ -86,6 +87,13 @@ contract FeeContract is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         emit Upgrade(newImplementation);
     }
 
+    /// @notice Cannot renounce ownership
+    /// @dev Override renounceOwnership() to revert, preventing accidental or malicious ownership
+    /// renunciation
+    function renounceOwnership() public virtual override onlyOwner {
```

Comment:
Do we currently have code to deploy a patch upgrade for the fee contract? If not, maybe a TODO/issue somewhere.